documents/motions/10.2022.0 - Committees and Teams.md

```diff
@@ -1,43 +1,43 @@
 2. All WCA Committees and Teams shall be formed for an indefinite period, unless otherwise decided by the WCA Board.
+   1. If a WCA Committee has fewer than 2 members it will not be considered constituted and its rights and duties will be undertaken by the WCA Board until 2 or more members are appointed. 
 3. Each WCA Committee and Team will have a direct relationship with at least one Director for management, communication, and escalation.
 4. Each WCA Committee and Team shall have one Leader.
-   1. In the case a Committee or Team does not have a leader it will be led by the WCA Board.
-   2. The Leader of every WCA Committee and Team is chosen by the WCA Board, based on a voting in the WCA Board with at least supermajority, after a request for applications among the Community.
-      1. Senior Delegates and Committee/Team Leaders must be provided with at least a one week time period to question, interview, and debate with the proposed candidate.
-      2. The choice of the new Committee/Team Leader must be confirmed in a vote among all the Senior Delegates and Committee/Team Leaders after the debate period has ended:
-         1. There must be an effective participation strictly above 50% of the total eligible voters for the vote to be considered valid.
-         2. There must be an option to abstain.
-         3. A majority of votes cast must be affirmative for the vote to pass.
-      3. After confirmation, the Committee/Team Leader will be appointed by the WCA Board.
-      4. In case there is no majority to confirm the selection, another candidate may be proposed by the WCA Board.
-   3. The Committee Leader is responsible for:
+   1. In the case a Committee or Team does not have a leader it will be led by the WCA Board
+   2. The Leader of every WCA Committee and Team is chosen by the WCA Board, based on a vote in the WCA Board with at least a supermajority, after a request for applications among the Community.
+   3. The WCA Board may appoint an interim Leader for a period of up to 3 months without a request for applications among the Community.
+         1. The selection of an interim Leader by the WCA Board is based on a vote in the WCA Board with at least a supermajority.
+         2. It is not possible to extend the term of an interim Leader.
+         3. The term of an interim Leader automatically ends once a Leader is appointed by the WCA Board following the process outlined in section 4.2.
+   4. The Committee Leader is responsible for:
       1. Making sure the Committee and the Committee Members are capable, equipped, and available for performing the duties of the Committee.
       2. Selecting, appointing, promoting, demoting, and removing Committee Members.
+         1. Approval of the WCA Board is required for the demotion of a Senior Member in cases where the Senior Member has not sent a letter of resignation. 
```

Comment:
Add in a line that a Committee Leader should / must consult with the WCA Board prior to recruiting new members.  This would avoid situations where there are upcoming changes that are not visible to the Team Lead.

Reply:
Maybe make it as a "should consult the WCA Board before making a public call for applicants". Beyond your concern Kerrie, I find the current system chaotic and I think it is a bad look when 3 teams have applications open close to each other. Best to spread out or bundle up. And give more certainty on when teams will next open up for applications!